app/mailers/review_mailer.rb

```diff
@@ -4,45 +4,34 @@ class ReviewMailer < ApplicationMailer
   def review_needed_email(draft)
     @draft = draft
     emails = draft.friend.remote_clinic_lawyers.map(&:email)
-    mail(to: emails,
-         body: review_needed_body(draft),
-         content_type: "text/html",
-         subject: "Review needed", )
+    @link_url = remote_clinic_friend_url(draft.friend)
```

Comment:
I wasn't sure about the need to include user here because I didn't see it being used in the controller.